spiketoolkit/comparison/sortingcomparison.py

```diff
@@ -153,6 +166,8 @@ def computeCounts(self):
             self._do_counting(verbose=False)
 
     def plotConfusionMatrix(self, xlabel=None, ylabel=None):
+        # Samuel EDIT
+        # This must be moved in spikewidget
```

Comment:
yes you can remove this. We can move it later from previous commits

Reply:
Before remove it do you want to copy/paste in spikewidget ?
Maybe wait to refactor spikewidget before ?